### PR TITLE
Configurable number of ticks in protvista-sequence

### PR DESCRIPTION
**Pull request by scheuerv**
### Description of changes
Adds a new attribute to protvista-sequence to configure the number of ticks.

### Tests / Styleguide
 - [x] Tests pass
 - [x] Styleguide

### CODE REVIEW

**Review by xwatkins — APPROVED** [review]
Could you please add an (or update one of the) example to `ProtvistaSequence.jsx`? Also please add to the doc under `/packages/protvista-sequence/README.md`

**Comment by xwatkins** [comment]
Are you able to merge it yourself now that it's been approved? If not, let me know and I'll hit the green button. Thank you!

**Comment by scheuerv** [comment]
> Are you able to merge it yourself now that it's been approved? If not, let me know and I'll hit the green button. Thank you!

No, unfortunately I don't have a write access so I can't merge it. Can you please merge it? Thank you.

**Pull request merged**
